deployment/CP03ISSM_Deploy.csv

```diff
@@ -181,22 +181,22 @@ AR-18,,AR-24,,CP03ISSM-SBC11-00-CPMENG000,6,1,2017-06-15T13:43:00,2017-11-01T14:
 AR-18,,AR-24,,CP03ISSM-SBD11-00-DCLENG000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGCON-BDCL11-50004,40.36706667,-70.88161667,,0,93,
 AR-18,,AR-24,,CP03ISSM-SBD11-01-MOPAK0000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGINS-MOPAK0-11839,40.36706667,-70.88161667,,0,93,
 AR-18,,AR-24,,CP03ISSM-SBD11-02-HYDGN0000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGINS-HYDGN0-49868,40.36706667,-70.88161667,,0,93,
-AR-18,,AR-24,,CP03ISSM-SBD11-06-METBKA000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGINS-METLGR-00033,40.36706667,-70.88161667,,-3,93,
+AR-18,,AR-24,,CP03ISSM-SBD11-06-METBKA000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGINS-METLGR-00033,40.36706667,-70.88161667,,0,93,
 AR-18,,AR-24,,CP03ISSM-SBD12-00-DCLENG000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGCON-BDCL12-50004,40.36706667,-70.88161667,,0,93,
 AR-18,,AR-24,,CP03ISSM-SBD12-03-HYDGN0000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGINS-HYDGN0-41692,40.36706667,-70.88161667,,0,93,
 AR-18,,AR-24,,CP03ISSM-SBD12-04-PCO2AA000,6,1,2017-06-15T13:43:00,2017-11-01T14:17:00,CGMCP-03ISSM-00006,,CGINS-PCO2AA-35260,40.36706667,-70.88161667,,0,93,
```

Comment:
Reviewed correction to DCL35 SN & clarification note, and deployment depths. Looks good.